src/auth/bearer.ts

```diff
@@ -0,0 +1,21 @@
+import { toText } from "../fetch/cors.js";
+
+// https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/WWW-Authenticate#directives_
+const TOKEN_STRINGS = "[A-Za-z0-9._~+/-]+=*";
+const PREFIX = "Bearer";
+const REALM = "sign";
+
+export const NoAuthorization = toText("Unauthorized", 400, new Headers({ "WWW-Authenticate": `${PREFIX} realm="${REALM}"` }));
+export const Unauthorized = toText("Unauthorized", 401, new Headers({ "WWW-Authenticate": `${PREFIX} error="invalid_token"` }));
+export const InvalidRequest = toText("Bad Request", 400, new Headers({ "WWW-Authenticate": `${PREFIX} error="invalid_request"` }));
```

Comment:
I've renamed this error response as `AuthInvalidRequest`

Just to not get confused with the other `BadRequest` from other invalid requests no-auth related

Reply:
Renamed it to `InvalidAuthRequest`.